cleanrl/sac_continuous_action.py

```diff
@@ -148,12 +201,16 @@ def to(self, device):
 class SoftQNetwork(nn.Module):
     def __init__(self):
         super(SoftQNetwork, self).__init__()
-        self.fc1 = nn.Linear(input_shape+output_shape, 120)
-        self.fc2 = nn.Linear(120, 84)
-        self.fc3 = nn.Linear(84, 1)
+        self.fc1 = nn.Linear(input_shape+output_shape, 256)
+        self.fc2 = nn.Linear(256, 128)
+        self.fc3 = nn.Linear(128, 1)
+        self.apply(weights_and_bias_init_)
 
     def forward(self, x, a):
-        x = preprocess_obs_fn(x)
+        if not isinstance(x, torch.Tensor):
```

Comment:
Consider looking at how it is done in https://github.com/vwxyzjn/cleanrl/blob/65d0e4a1c3be60fd590b30108869b764990a2cb3/cleanrl/td3_continuous_action.py#L143